core/src/main/java/com/yahoo/tagchowder/Parser.java

```diff
@@ -280,6 +287,7 @@
         theFeatures.put(RESTART_ELEMENTS_FEATURE, truthValue(DEFAULT_RESTART_ELEMENTS));
         theFeatures.put(IGNORABLE_WHITESPACE_FEATURE, truthValue(DEFAULT_IGNORABLE_WHITESPACE));
         theFeatures.put(CDATA_ELEMENTS_FEATURE, truthValue(DEFAULT_CDATA_ELEMENTS));
+        theFeatures.put(AMP_VALIDATION_FEATURE, truthValue(DEFAULT_AMP_VALIDATION));
```

Comment:
add one UT for amp feature
1) use getFeature() funtion to check the default value of AMP_VALIDATION_FEATURE is false
2) call setFeature() to set AMP_VALIDATION_FEATURE, call getFeature() again, it should return true

Reply:
Added